app/models/policy_group.rb

```diff
@@ -73,6 +85,6 @@ def summary_or_name
              description: :summary
 
   def search_link
-    Whitehall.url_maker.policy_group_path(slug)
+    public_path
```

Comment:
As the `search_link` method is only used once, is it possible to inline `:public_path` on line 83 rather than defining this method?